docs/apm/traces/working-with-tracing-data/service-map.md

```diff
@@ -1,64 +1,96 @@
 
-![service map gear icon for settings.png](/img/traces/service-map-gear-icon-for-settings.png)
+### Settings
 
-### Anomaly Detection
+KPIs listed in the table reflect the previous 15 minutes of data. Click on any service opens the Entities panel supporting further drill-down.
 
-Automatic anomaly detection is enabled by default. A service is displayed in a red circle if at least one of the KPIs (latency, errors, requests) has an anomaly (a spike for latency and errors or dip in the case of requests) in the last 15 minutes AND the anomaly was not present 7 days ago at the same time. In the settings menu, this default option is called **Worst Case**. You can explicitly choose one of the KPIs instead.
+* Filter and sort your KPI data using the filters and column headers <br/>![services-list-display](/img/traces/services-list-filters.png)
+* Display and hide columns using **Display Settings** <br/>![services-list-display](/img/traces/services-list-display.png)
+* Configure anomalies in **Settings**<br/> <img src={useBaseUrl('img/traces/services-list-settings.png')} alt="services-list-settings" width="400"/>
 
-The standard metric outlier operator is used to detect this. You can adjust the sensitivity and **learning window** of the outlier detection in the settings menu to make the logic more or less dependent on occasional spikes. 
 
-![auto service map settings.png](/img/traces/auto-service-map-settings.png)
+## Services Map view
 
-Manual anomaly detection settings allow you to specify different detection settings for each KPI. Click the toggle switch to activate these settings on and off. You can explicitly toggle each KPI as desired.
+The **Services Map** view visually maps out your application environment, giving you a greater understanding of your application architecture, hierarchy, and dependencies between monitored microservices. Health and load of each microservice is reflected in size and color to help you spot potential problems and bottlenecks in your application infrastructure.
+
+Maps are built automatically from distributed tracing data in real time as it arrives in Sumo Logic.
+
+<img src={useBaseUrl('img/traces/services-map.png')} alt="servicemap" />
+
+The Services Map provides the following features:
+
+* Zoom in and out on the map
+* Hover over a service to see its connections to other services and the last 15 minutes of activity in terms of latency, requests, and errors
+* Click on a service to open the Entity Inspector with the ability to drill down to traces, metrics, and the service dashboard
+* Filter by application (if your tracing data has the `application=[app-name]` tag)
+* Search for a service by name<br/> ![service map april 2021.png](/img/traces/service-map-example.png)
```

Comment:
Cant find that image, not sure why. On my local yarn instance this shows as outdated with no environment filter.

Reply:
https://github.com/SumoLogic/sumologic-documentation/pull/1313#discussion_r1072662107